smart_contracts/contracts_as/client/undelegate/assembly/index.ts

```diff
@@ -53,6 +53,7 @@ export function call(): void {
     }
     let amount = amountResult.value;
 
+
```

Comment:
That's probably a typo?

Reply:
You mean the duplicated empty line, right? If so, I removed it here: 556f02b279a61e2fbacfe6df40165e3369dcda61